inginious/frontend/plugins/contact_page/pages/constants.py

```diff
@@ -46,3 +46,8 @@ def get_subject_new_course_id():
 def get_use_minify():
     """ return a boolean to define if use minified files """
     return _use_minified
+
+
+def contact_us_option_hook():
+    return "<li><a href='https://juezun.github.io/' class='navbar-link' target='_blank'>" + \
```

Comment:
This item of the UNCode nav element has the same reference as the "About us" page. Not sure if this element needs to reference to the contact page.

Reply:
Good catch, thanks!